controllers/admin/AdminModulesController.php

```diff
@@ -1508,4 +1527,21 @@ public function ajaxProcessGetModuleReadMoreView()
             'body' => $this->context->smarty->fetch('controllers/modules/readmore-body.tpl')
         )));
     }
+
+    /**
+     * Check if the module uses the new translation system
+     * @param $moduleName
+     * @return bool
+     */
+    private function checkTranslationSystem($moduleName)
```

Comment:
Can you move this in a service because it's the same code ? and this https://github.com/PrestaShop/PrestaShop/blob/e6c49b4c3efc048260a44b40b4a5dd2b08239fc4/controllers/admin/AdminTranslationsController.php#L261 too ?